src/hoc/selectable-enhance.js

```diff
@@ -76,12 +76,20 @@ export const SelectableContainerEnhance = (Component) => {
           key: this._keyIndex,
           style: mergedChildrenStyles,
           nestedItems: child.props.nestedItems.map((child) => this._extendChild(child, styles, selectedItemStyle)),
+          initiallyOpen: child.props.nestedItems.reduce(this._hasSelectedDescendant, false),
```

Comment:
Yeah, I guess so - I was trying to avoid adding another prop, and even though it's a breaking change, it feels like it will be universally beneficial (when would you ever want selected items not displayed?). 

However, easing into it is perhaps the better approach. If we have it disabled by default with a prop to enable it, we can always flip the default in the future as a breaking change.

As for prop-name, how about `initallyOpen`, as per `ListItem`, but rather than a `bool`, it could take an `enum` of `['true' | 'false' | 'auto']`, where `true` opens all the items, `false` is current behaviour (none open), and `auto` is the new feature?